index.html

```diff
@@ -349,6 +361,7 @@ <h3 class="member-sub-heading">Devon Lane</h3>
         </section>
         <section class="partners">
             <h2 class="partners heading"><span class="colored">BINABOX</span> PARTNERS</h2>

```

Comment:
The class `.partners` seems to be applied to the entire section. For better clarity, consider using a more specific class, like `.partners-heading`, for the section's title or heading.